<launch>
  <arg name="rosbag" doc="rosbag file path" />
  <rosparam param="/use_sim_time">
    true
  </rosparam>
  
  <node name="rosbag_play" pkg="rosbag" type="play"
        args="$(arg rosbag) -l --clock"/>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find multi_device_view)/config/d405_test.rviz"/>

  <node name="republish" pkg="image_transport" type="republish" args="compressed raw">
    <remap from="in" to="/camera/color/image_rect_color" />
    <remap from="out" to="/camera/color/image_rect_color" />
  </node>

  <node name="depth_image_republish" pkg="image_transport" type="republish" args="compressedDepth raw">
    <remap from="in" to="/camera/depth/image_rect_raw" />
    <remap from="out" to="/camera/depth/image_rect_raw" />
  </node>
  
</launch>
